### ISSUE TITLE
"Print All Assigned" --> 500 Server Error 

### ISSUE THREAD

**Issue by SpitFireRSA** [grokability/snipe-it#15760]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Hi Folks

When clicking the "Print All Assinged" button, I am getting a "500 | Server Error" response.

Please see below:

Docker Image: linuxserver/snipe-it:7.0.13
Server Version:
![image](https://github.com/user-attachments/assets/71ccc594-d5d2-44b8-bb46-74139b2873b8)

Debug:
![image](https://github.com/user-attachments/assets/d2f055ad-c90e-4fc6-8bce-488b199bd5e6)

### Reproduction steps

Log into
Navigate to the "People" section
Use the Search function to find the employee.
Click on the employees name to open their profile.
Click on the "Print All Assigned" button and save the file.

### Expected behavior

Provide a web page with the assigned assets to then print to PDF.

### Snipe-IT Version

v7.0.13 - build 15514 (master)

### Operating System

Docker

### Web Server

NGINX?

### PHP Version

8.3.12

### Operating System

OSX

### Browser

Google Chrome

### Version

Version 130.0.6723.60 (Official Build) (arm64)

### Error messages

```shell
Undefined variable $show_user

resources/views/users/print.blade.php: 5 require

<!doctype html>

<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">

<head>

    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

    @if (count($users) === 1)

        <title>{{ trans('general.assigned_to', ['name' => $users[0]->present()->fullName()]) }} - {{ date('Y-m-d H:i', time()) }}</title>

    @else

        <title>{{ trans('admin/users/general.print_assigned') }} - {{ date('Y-m-d H:i', time()) }}</title>

    @endisset



    <link rel="shortcut icon" type="image/ico" href="{{ ($snipeSettings) && ($snipeSettings->favicon!='') ?  Storage::disk('public')->url(e($snipeSettings->favicon)) : config('app.url').'/favicon.ico' }}">



    <link rel="stylesheet" href="{{ url(mix('css/dist/bootstrap-table.css')) }}">



    {{-- stylesheets --}}

    <link rel="stylesheet" href="{{ url(mix('css/dist/all.css')) }}">



    <script nonce="{{ csrf_token() }}">

        window.snipeit = {

            settings: {

                "per_page": 50

            }

        };

    </script>



    <style>

        body {

            font-family: "Arial, Helvetica", sans-serif;

            padding: 20px;

        }

=====

Routing
Controller: App\Http\Controllers\Users\UsersController@printInventory
Route Name: users.print

Views: resources/views/users/print.blade.php
```

**Comment by snipe** [comment]
I can't reproduce this locally. Are you able to reproduce it on the demo? https://demo.snipeitapp.com

<blockquote><img src="https://demo.snipeitapp.com/favicon.ico" width="48" align="right"><div><strong><a href="https://demo.snipeitapp.com/">Snipe-IT Asset Management Demo</a></strong></div></blockquote>

**Comment by SpitFireRSA** [comment]
Thanks for getting back to me 😄  ! Hmm.  It Works as expected on the demo.

**Comment by SpitFireRSA** [comment]
Might be worth mentioning that I am doing this action as the SuperAdmin.  I recall testing this with a "normal" user and confirming that it works.  This was in the previous version.  let me assign an asset and recheck

**Comment by SpitFireRSA** [comment]
Aha - so it works with my normal user, just not with my super-admin account:

![image](https://github.com/user-attachments/assets/6a5d2563-a2b9-498f-9191-82069a283882)

**Comment by snipe** [comment]
That kinda doesn't make sense - I'd expect the opposite, if anything. Can you upgrade to latest on master?

**Comment by snipe** [comment]
(Like, I *believe* you're seeing what you're describing - just from a code perspective it doesn't make sense.)

**Comment by SpitFireRSA** [comment]
For sure - I am also confused.  I am currently using docker (linuxserver) and using the latest image tag available (that should match master - 7.0.13).

**Comment by SpitFireRSA** [comment]
This isn't mission critical for me, because I have delegated admins that will pull these reports when users offboard.  Just logging here as I noted it when writing the docs today.

**Comment by marcusmoore** [comment]
I touched code around this recently. I'll check it out.

**Comment by marcusmoore** [comment]
I can confirm there is a bug (similar but not the exact same error) on the tagged version of v7.0.13 that is not currently present in the master branch. We have some changes there that haven't been tagged for release yet. 

@SpitFireRSA this should be fixed for you in v7.0.14 when it is tagged and released. I'm going to close this but please ping me if you still have an issue after updating to v7.0.14

**Comment by SpitFireRSA** [comment]
Hey @marcusmoore & @snipe 

Thank you very much for your prompt assistance 🙏  Appreciated!
